composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "goalgorilla/open_social",
+    "name": "open-social-eci/open_social",
```

Comment:
Hi @ksis,

We should not commit this line to the repo (because it is an override for the Fork). Not sure what the best way is to fix it for now, maybe do another commit in the branch that sets the line back